nautobot_floor_plan/tests/test_api_views.py

```diff
@@ -46,31 +46,59 @@ class FloorPlanTileAPIViewTest(APIViewTestCases.APIViewTestCase):
 
     model = models.FloorPlanTile
     brief_fields = ["display", "id", "url", "x_origin", "x_size", "y_origin", "y_size"]
-    choices_fields = ["rack_orientation"]
+    choices_fields = ["rack_orientation", "allocation_type"]
     # TODO choices_fields = ["rack_orientation", "status"]
```

Comment:
Can remove this line I think.

Reply:
Unable to remove 49. Received the following error when running unittests
```
 All field names of choice fields for a given model serializer need to be manually added to self.choices_fields. If this is already the case, perhaps the serializer is implemented incorrectly?
```

Reply:
Sorry, I was actually referring to the commented-out line 50.